app/controllers/users_controller.rb

```diff
@@ -1,9 +1,17 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  before_action :logged_in_user, except: %i(new create)
+  before_action :load_user, except: %i(new create index)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: :destroy
+
+  def index
+    @pagy, @users = pagy User.sorted_by_name, items: Settings["PERPAGE_5"]
+  end
+
   def show
```

Comment:
```suggestion
  def show; end
```
đã có load user làm hết rồi, ngồi chơi xơi nước thôi cu